client/store.js

```diff
@@ -55,6 +55,16 @@ store.getCurrentAccountId = function() {
     return data.currentAccountId;
 }
 
+store.getDefaultAccountId = function() {
+    return data.settings.get('defaultAccountId');
+}
+
+store.setDefaultAccountId = function(id) {
+    //Do not use store.changeSetting so that the change is immediately visible
```

Comment:
And remove it from here : https://github.com/nicofrand/kresus/blob/defaultAccount/client/store.js#L1123, right ?